rimage/image_file.go

```diff
@@ -350,6 +379,18 @@ func fastConvertRGBA(dst *Image, src *image.RGBA) {
 	}
 }
 
+// ConvertToRGBA converts an rimage.Image type image to image.RGBA.
+func ConvertToRGBA(dst *image.RGBA, src *Image) {
```

Comment:
This may be for another PR, but we can probably speed this up by getting the source array all at once and inline some code and avoid calling a lot of functions per pixel. That being said, we should benchmark that bit but I imagine it could result in some single digit % boost.